src/css/bs-stepper.css

```diff
@@ -1,198 +1,117 @@
 /*!
  * bsStepper v1.3.0 (https://github.com/Johann-S/bs-stepper)
- * Copyright 2018 Johann-S <johann.servoire@gmail.com>
+ * Copyright 2018-2019 Johann-S <johann.servoire@gmail.com>
  * Licensed under MIT (https://github.com/Johann-S/bs-stepper/blob/master/LICENSE)
  */
 
-.bs-stepper {
-  display: block;
-}
-
-.bs-stepper.vertical {
-  display: flex;
-}
-
-.bs-stepper .bs-stepper-header {
-  white-space: nowrap;
-  display: flex;
-  align-items: center;
-}
-
-.bs-stepper.vertical .bs-stepper-header {
-  flex-direction: column;
-  align-items: flex-start;
-}
-
-.bs-stepper .bs-stepper-content {
-  overflow: hidden;
-  padding: 0 24px 24px 24px;
-}
-
-.bs-stepper.vertical .bs-stepper-content {
-  padding: 0 0 0 48px;
-}
-
-.bs-stepper .step {
-  display: flex;
-  height: 72px;
-  overflow: hidden;
+.bs-stepper-btn {
```

Comment:
I think this class isn't usefull because there is `.step-trigger` which is already on the same elements